taskcluster/fenix_taskgraph/transforms/browsertime.py

```diff
@@ -124,3 +124,32 @@ def fill_email_data(config, tasks):
             email["subject"] = email["subject"].format(**format_kwargs)
 
         yield task
+
+
+@transforms.add
+def filter_ytp_tests(config, tasks):
+    """Youtube-playback AV1 and VP9 tests should run on P2 only"""
+    for task in tasks:
+        test_platform = task['treeherder']['platform']
+        test_name = task['name']
+        if "AV1" in test_name or "VP9" in test_name:
+            if 'android-hw-p2' in test_platform:
+                yield task
+            else:
+                continue
+
+        # Bug 1668537 The following tests are failing intermittently
+        # and are disabled until the error is fixed
+        if "WIDEVINE-H264" in test_name or "WIDEVINE-H264-HFR" in test_name:
+            if 'android-hw-g5' in test_platform:
+                yield task
+            else:
+                continue
+
+        if "WIDEVINE-HFR" in test_name or "WIDEVINE-VP9-SFR" in test_name:
```

Comment:
Same as above.

Reply:
Actually, you can combine this condition with the one above by checking if `widevine` is in the test name.